_toc.yml

```diff
@@ -577,6 +577,8 @@ entries:
         title: HowTo
         entries:
           - file: docs/products/flink/howto/create-integration
+          - file: docs/products/flink/howto/manage-flink-applications
+            title: Manage Flink applications
```

Comment:
Should the title be Manage Aiven for Apache Flink applications (since it's not native flink feature)?